tests/test_app.py

```diff
@@ -40,6 +40,24 @@ def test_compile_all(settings):
         assert '"Hello world!": "Bonjour \\u00e0 tous !"' in content
 
 
+def test_compile_no_use_i18n(settings):
+    """Tests compilation when `USE_I18N = False`.
+
+    In this scenario, only the default locale (`settings.LANGUAGE_CODE`) is
+    processed, which is `en-us` for a default project.
+    """
+    settings.USE_I18N = False
```

Comment:
What do you think about setting explicitly a `LANGUAGE_CODE` so we can verify it's the right one that get processed?